Firmware/mmu2.cpp

```diff
@@ -1048,4 +1012,24 @@ void MMU2::OnMMUProgressMsgSame(ProgressCode pc){
     }
 }
 
+//void MMU2::LogErrorEvent(const char *msg) {
+//    MMU2_ERROR_MSG(msg);
+//    SERIAL_ECHOLN();
+//}
+
+void MMU2::LogErrorEvent_P(const char *msg_P) {
+    MMU2_ERROR_MSGRPGM(msg_P);
+    SERIAL_ECHOLN();
+}
+
+//void MMU2::LogEchoEvent(const char *msg) {
+//    MMU2_ECHO_MSG(msg);
+//    SERIAL_ECHOLN();
+//}
+
+void MMU2::LogEchoEvent_P(const char *msg_P) {
+    MMU2_ECHO_MSGRPGM(msg_P);
+    SERIAL_ECHOLN();
+}
+
```

Comment:
We are we re-defining these functions? I see they're present in `mmu2_log.cpp`, shouldn't we use those?